gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileExporter.java

```diff
@@ -95,9 +95,9 @@ public String saveVoltageMonitoringResultInJsonZip(VoltageMonitoringResult resul
                                                        String fileType) {
         MemDataSource memDataSource = new MemDataSource();
         try (OutputStream os = memDataSource.newOutputStream(targetName, false)) {
-            VoltageCheckResult voltageCheckResult = voltageResultMapper.mapVoltageResult(result);
+            Object resultToWrite = result != null ? voltageResultMapper.mapVoltageResult(result) : new FailureVoltageCheckResult();
             ObjectWriter objectWriter = new ObjectMapper().writer().withDefaultPrettyPrinter();
```

Comment:
I don't see any problem with this code. 
However, we might want to consider to break down this method into separate method (one for conversion, one for zipping and one for uploading). Maybe not in this PR though

Reply:
ok I understand.